props/nested-components.md

```diff
@@ -0,0 +1,195 @@
+# Nested Components
+
+## Goals
+
+In order to increase the complexity of our React webapps, we should practice nesting components.
```

Comment:
Increasing the complexity isn't the goal. But our apps _will_ become more complex, and nesting components will help us manage that.
```suggestion
In order to manage the increasing complexity of our React webapps, we should practice nesting components.
```